src/v0/analytics.ts

```diff
@@ -54,6 +54,8 @@ export interface VoucherOptions {
   valid_until?: string
   comment?: string
   q?: string
+  amount?: string | number
```

Comment:
this looks like it could easily turn into a nightmare (i.e. when modifying amount using numbers), but in this context it's probably fine. Don't know what consequences there would be when enforcing only `number` here